config/spinnaker.yml

```diff
@@ -212,9 +212,14 @@ providers:
     # set enabled and provide primary credentials for deploying.
     # Enabling azure is independent of other providers.
     enabled: false
-    defaultRegion: West US
+    defaultRegion: westus
     primaryCredentials:
       name: my-azure-account
+      clientId:
+      appKey:
+      tenantId:
+      subscriptionId:
+    # To set azure credentials, enter values for: clientId, appKey, tenantId, subscriptionId.  See spinnaker documentation - cloud provider setup for instructions on how to retrieve these values.
```

Comment:
I was trying to keep 80 columns in these files if practical.
If you could link to the docs here that would be better (the URL could go beyond 80). I'd also add the link to the default-spinnaker-local because that is the point of the typical user interaction. The expectation is to start in spinnaker-local and not need to look here for common needs.